src/org/mangui/hls/loader/AltAudioLevelLoader.as

```diff
@@ -125,6 +125,10 @@ package org.mangui.hls.loader {
 
         /** When audio track switch occurs, assess the need of loading audio level playlist **/
         private function _audioTrackSwitchHandler(event : HLSEvent) : void {
+            if (_hls.audioTracks.length == 0) {
```

Comment:
Are there any tests already in place for this file?  If so it might not be bad to add a test case.

Reply:
there are not

Reply:
👍